packages/react/core/README.md

```diff
@@ -0,0 +1,63 @@
+# `@wallet-standard/react-core`
+
+This package provides React hooks for using Wallet Standard wallets, accounts, and features.
+
+## Types
+
+### `ReactWallet`
+
+This type represents a `Wallet` in a React application. It acts as a &lsquo;handle&rsquo; to the underlying `Wallet` instance in the Wallet Standard registry, and contains a subset of its properties.
+
+You can pass objects of this type around your application as props or state, inspect its `chains` and `features`, and enumerate the `accounts` for which your application has been granted authorization to use.
+
+### `ReactWalletAccount`
+
+This type represents a `WalletAccount` in a React application. It acts as a &lsquo;handle&rsquo; to the underlying `WalletAccount` instance in the Wallet Standard registry, and contains a subset of its properties.
+
+You can pass objects of this type around your application as props or state, inspect its `chains` and `features`, and enumerate the `accounts` for which your application has been granted authorization to use.
```

Comment:
It looks like this line is copied from `ReactWallet` and wouldn't apply to `ReactWalletAccount`